### DISCUSSION TITLE
Question 4: bias

### DISCUSSION THREAD

**Discussion by mybirth0407** [Boostcamp-AI-Tech-1-15/ddobokki_log#32] [2. DL Basic]
Neural nets 에서 bias는 왜 있는 걸까요?

**Comment by 20180707jun** [comment]
예를 들어, y = ax+b와 같은 함수가 있을 때, b가 없어지게 되면 함수는 필연적으로 (0, 0)을 지나게됩니다. 하지만 b가 있다면 (0, 0)을 필연적으로 지나지 않아도 됩니다. 다른말로 하게 되면 b는 activation function의 평행이동을 시켜주는 역할을 하게 되고, 적절한 평행이동은 저희가 예측하고자 하는 함수를 더 fit하게 예측할 수 있게 해줍니다.
맞나요...?

**Reply by mybirth0407** [reply]
네 준영님 맞습니다! 간단하게 bias가 없으면 input vector의 element가 0인 경우에, 반드시 (x, wx) = (0, 0) 을 지나기 때문에, 데이터 전체에 대해 fit해지기가 어려워요! 그런 부분을 해결하고자 bias가 있는 것입니다!
